src/components/PokeCard.js

```diff
@@ -66,22 +69,15 @@ const PokeCard = ({
           <span> #{dexNo}</span>
         </li>
         <li>{typeTags}</li>
-        <li>
-          <button
-            onClick={() => {
-              setModalIsOpen(true);
-              triggerModalData();
-            }}>
-            Open
-          </button>
-        </li>
       </ul>
-      <Modal
+    </div>
+    <Modal
+        onAfterOpen={triggerModalData}
         className="modalWindow"
         isOpen={modalIsOpen}
         moveSet={moveSet}
         onRequestClose={() => {
-          return setModalIsOpen(false);
+          setModalIsOpen(false);
         }}>
         <CustomModal
```

Comment:
Is this a modal inside of a modal or am I missing something? Either way, interesting.

Reply:
I think it might be part of a package they might be using which requires it.

Reply:
It's just a confusing name to give that component, which we'll fix in refactor later.